cves/kernel/CVE-2015-1573.yml

```diff
@@ -219,8 +237,11 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: ["netfilter", "nftables"]
+  note: |
+    This vulnerability is more specifically related to the nftables subsystem,
```

Comment:
A suggestion for clarity but just another nitpick: This vulnerability is specifically related to the nftables subsystem, even though the file containing the vulnerable code is located in the netfilter directory

Reply:
Updated